### PR TITLE
Added Style/SpaceInsideBrackets Rubocop cop

### PR DESCRIPTION
**Pull request by divyanshumehta**
The Style/SpaceInsideBrackets cop has been enabled to check for useless spaces inside brackets. Fixes Issue #1497 .

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -147,7 +147,7 @@ def concurrent_events(event)
   end
 
   def speaker_links(event)
-    event.speakers.map{ |speaker| link_to speaker.name, admin_user_path(speaker) }.join(', ').html_safe
+    event.speakers.map{ |speaker| link_to speaker.name, admin_user_path(speaker) }.join(', ').safe_join
```
Comment (Ana06) [inline app/helpers/application_helper.rb]: This is unrelated to the commit, I guess Rubocop was failing and that's why you fixed it, but please add it in a different commit. :wink:

**Comment by Ana06** [comment]
And please rebase!!!! :slightly_smiling_face:

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (divyanshumehta) [inline app/helpers/application_helper.rb]: this creates a rubcop error in the travis CI

**Comment by divyanshumehta** [comment]
@Ana06 why do I need to rebsae the commits? I couldn't get it.

**Comment by Ana06** [comment]
@divyanshumehta 

> @Ana06 why do I need to rebsae the commits? I couldn't get it.

Because there are merge conflicts and the PR can not be merged when there are merge conflicts. https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line

**Comment by divyanshumehta** [comment]
I dont see merge conflicts though it says `merging cannot be done until all requested changes are addressed`

**Comment by divyanshumehta** [comment]
@Ana06 all test pass now and evrything is fixed.

**Pull request merged**